### ISSUE TITLE
Add installation guideline in docs

### ISSUE THREAD

**Issue by gabiudrescu** [Monofony/Monofony#154]
Steps to reproduce:
 - search on google „monofony”  
 - open documentation https://monofony.readthedocs.io/en/latest/ or deprecated monofony repo
 - ask yourself how to install this

what happens:
- you are a bit confused and left with a bad feeling about this

what should happen:
 - you should see in the first page from docs the installation guideline

if approved, I can provide PR to fix this.

**Comment by loic425** [comment]
Yes, we have to move the installation guide into the documentation.

**Comment by oliver-schulz** [comment]
@gabiudrescu also take a look at the sylius documentation. Basically monofony is sylius without the eCommerce taste. 

@loic425 i really like this packages and was able to build an app in about a week.
